src/content/docs/installation/installation_t2macbook.mdx

```diff
@@ -3,98 +3,120 @@ title: T2 MacBook Installation
+    *   Restart your Mac and hold the `Option (⌥)` key immediately after powering on.
+    *   Select the CachyOS USB drive (usually labeled as "EFI Boot").
+2.  **(Optional) Enable Wi-Fi in the Live Environment:**
+    *   If you need Wi-Fi during installation and don't have Ethernet/tethering, open the terminal once the CachyOS live environment loads.
+    *   Run these commands to copy the firmware from the EFI partition (prepared in Step 1.3) and configure the network manager:
+        ```sh
+        # Mount the EFI partition (usually nvme0n1p1 on T2 Macs)
+        sudo mkdir -p /tmp/apple-wifi-efi
+        sudo mount /dev/nvme0n1p1 /tmp/apple-wifi-efi
+
+        # Copy firmware from EFI to the live environment
+        bash /tmp/apple-wifi-efi/firmware.sh get_from_efi
+        sudo umount /tmp/apple-wifi-efi
+
+        # Configure NetworkManager to use iwd backend
+        cat <<EOF | sudo tee /etc/NetworkManager/conf.d/wifi_backend.conf
+        [device]
+        wifi.backend=iwd
+        EOF
+        sudo systemctl restart NetworkManager
+        ```
+    *   You should now be able to connect to Wi-Fi networks using the network applet in the system tray.
+3.  **Run the CachyOS Installer:**
+    *   Launch the CachyOS installer from the desktop or application menu.
+    *   Follow the standard installation procedure, referring to the [Installation on Root](/installation/installation_on_root) guide.
+    *   When partitioning, select the free space created earlier. Let the installer handle formatting.
+        :::caution[Partitioning Caution]
+        Be **very careful** during the partitioning step in the installer. Ensure you are selecting the correct free space you prepared and **do not** accidentally delete or format your existing macOS partition, especially if you intend to dual-boot. Double-check your selections before proceeding.
+        :::
+    *   CachyOS Hardware Detection ([CHWD](/features/chwd)) should automatically apply necessary T2-specific boot parameters and configurations during installation.
 
-    <Tabs>
-
-    <TabItem label="using curl">
-
-    ```sh
-    curl https://mirror.funami.tech/arch-mact2/os/x86_64/apple-bcm-firmware-14.0-1-any.pkg.tar.zst -o apple-bcm-firmware-14.0-1-any.pkg.tar.zst
-    ```
+</Steps>
 
-    </TabItem>
-    <TabItem label="using wget">
+## Post-Installation Steps
 
-    ```sh
-    wget https://mirror.funami.tech/arch-mact2/os/x86_64/apple-bcm-firmware-14.0-1-any.pkg.tar.zst
-    ```
-    > If `wget` is unavailable, install it with `sudo pacman -S wget`
+After the installation is complete and you have rebooted into your new CachyOS system:
 
-    </TabItem>
-    </Tabs>
+<Steps>
 
-2. Install firmware package
-    ```sh
-    sudo pacman -U apple-bcm-firmware-14.0-1-any.pkg.tar.zst
-    ```
+1.  **Install Wi-Fi Firmware Permanently:**
+    *   The Wi-Fi firmware needs to be properly installed on your system. Connect to the internet (Ethernet, USB tethering, or repeat Step 2.2 if necessary).
```

Comment:
I think this is better;

If you have followed Step 2.2 to get firmware in the live ISO, you can simply follow it again. Otherwise connect to the internet (Ethernet.....) and follow the steps below: